nearlib/test/integration.test.js

```diff
@@ -147,9 +124,29 @@ const callUntilConditionIsMet = async (functToPoll, condition, description, maxR
                 fail('exceeded number of retries for ' + description + ". Last error " + e.toString());
             }
         }
+        await sleep(500);
     }
+    fail('exceeded number of retries for ' + description);
 };
 
+const waitForTransactionToComplete = async (submitTransactionResult) => {
+    expect(submitTransactionResult.hash).not.toBeFalsy();
+    console.log("Waiting for transaction" + submitTransactionResult.hash);
+    await callUntilConditionIsMet(
+        async () => { return await nearClient.getTransactionStatus(submitTransactionResult.hash); },
+        (response) => {
```

Comment:
I would write this whole block as:
`response => response.status == 'Completed`

Reply:
I want the log though